mscore/importmidi/importmidi.cpp

```diff
@@ -265,7 +265,7 @@ void MTrack::processMeta(int tick, const MidiEvent& mm)
             qDebug("processMeta: no staff");
             return;
             }
-      const uchar* data = (uchar*)mm.edata();
```

Comment:
note: I removed this (uchar*) because it is superfluous, since both edata() and data are uchar* already.